Assets/Scripts/Utilities/Settings.cs

```diff
@@ -245,7 +245,7 @@ private static void LoadSettings()
         XmlDocument xDoc = new XmlDocument();
         xDoc.LoadXml(furnitureXmlText);
         Debug.ULogChannel("Settings", "Loaded settings");
-        Debug.Log(xDoc.InnerText); // Uber Logger doesn't handle multilines.
+        Debug.ULogChannel("Settings", xDoc.InnerText); 
```

Comment:
This was purposely kept as a Debug.Log because , as the comment says, you cant read the full output in uberlogger. Technically, you can still see it in the normal log since uber passes messages to it, but its not very helpful to have that line in uberlogger.